app/models/application_record.rb

```diff
@@ -19,6 +19,11 @@ class ApplicationRecord < ActiveRecord::Base
   scope :newest_first, -> { order 'created_at desc' }
   scope :between,
         -> (start_date, end_date) { where created_at: start_date..end_date }
+  scope :in_department,
+        lambda { |department_id|
```

Comment:
I know Rubocop corrects `->` to `lambda`, but I disagree. We can change this one back to look like the other AR scopes.